ddp-workspace/projects/ddp-singular/playwright-e2e/tests/contact-us/contact-us-page-visual.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { expect, test } from '@playwright/test';
+import { fillSitePassword, goToAboutUs, goToPath } from 'tests/nav';
+
+/**
+ * Functional tests for the Home page
+ */
+test.describe('Home page', () => {
+  test.beforeEach(async ({ page }) => {
```

Comment:
Not sure but maybe, it would be enough to run that pre-conditions once using `beforeAll`?

Reply:
`beforeAll` is not right for this purpose. `beforeAll` runs (one time) before start of all tests. Each test will launch a new browser and close it when done.